components/brave_wallet/browser/solana_instruction.h

```diff
@@ -29,9 +32,16 @@ class SolanaInstruction {
   SolanaInstruction(const std::string& program_id,
                     std::vector<SolanaAccountMeta>&& accounts,
                     const std::vector<uint8_t>& data);
+  SolanaInstruction(const std::string& program_id,
+                    std::vector<SolanaAccountMeta>&& accounts,
+                    const std::vector<uint8_t>& data,
+                    absl::optional<SolanaInstructionDecodedData> decoded_data);
   ~SolanaInstruction();
 
   SolanaInstruction(const SolanaInstruction&);
+  SolanaInstruction(SolanaInstruction&&);
+  SolanaInstruction& operator=(const SolanaInstruction&);
+  SolanaInstruction& operator=(SolanaInstruction&&);
```

Comment:
We can remove these because it is now default behavior, both copyable and movable. So there is not need to explicitly declare all four. Same applies to `SolanaInstructionDecodedData`
`Explicitly declaring or deleting all four copy/move operations is permitted, but not required` from
https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types

Reply:
Unfortunately we can't, `error: [chromium-style] Complex class/struct needs an explicit out-of-line copy constructor.` for both.